tidb-cloud/tidb-cloud-org-sso-authentication.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Organization SSO Authentication
+summary: Learn how to log in to the TiDB Cloud console via your customized organization authentication.
+---
+
+# Organization SSO Authentication
+
+Single Sign-On (SSO) is an authentication scheme that enables members in your TiDB Cloud [organization](/tidb-cloud/tidb-cloud-glossary.md#organization) to log into TiDB Cloud using identities from an identity provider (IdP) instead of email addresses and passwords.
```

Comment:
```suggestion
Single Sign-On (SSO) is an authentication scheme that enables members in your TiDB Cloud [organization](/tidb-cloud/tidb-cloud-glossary.md#organization) to log in to TiDB Cloud using identities from an identity provider (IdP) instead of email addresses and passwords.
```